sql/range_cut.go

```diff
@@ -157,6 +159,11 @@ func (a Above) String() string {
 	return fmt.Sprintf("Above[%v]", a.Key)
 }
 
+// DebugString implements RangeCut.
+func (a Above) DebugString() string {
+	return fmt.Sprintf(">%v", a.Key)
```

Comment:
I get why these were added, but considering ranges have the concept of greater than and less than, these will cause more confusion than clarity. You'd have to look at the `DebugString` functions to know that this isn't _greater than key_, but is instead _above key_. The original strings are much more clear with that, so I think these should be removed, or internally just call `a.String()`.